exporter/collector/logs.go

```diff
@@ -531,15 +531,24 @@ func (l logMapper) logToSplitEntries(
 	return []logging.Entry{entry}, nil
 }
 
-func parseEntryPayload(logBody pcommon.Value, maxEntrySize int) (interface{}, int, error) {
+func (l logMapper) parseEntryPayload(logBody pcommon.Value, maxEntrySize int) (interface{}, int, error) {
 	if len(logBody.AsString()) == 0 {
 		return nil, 0, nil
 	}
+
+	// number of payloads calculation, used when returning a string payload
+	payloads := int(math.Ceil(float64(len([]byte(logBody.AsString()))) / float64(maxEntrySize)))
+
 	switch logBody.Type() {
 	case pcommon.ValueTypeBytes:
+		var verify map[string]interface{}
+		if err := json.Unmarshal(logBody.Bytes().AsRaw(), &verify); err != nil {
```

Comment:
If we always sent the log body as a string, would that break users?  Or would the logging API still display it correctly?

Reply:
if it can be parsed as an object, we want to send it as an object (protoPayload/jsonPayload). I don't think we want to send everything as a string